libs/vre/shared/app-resource-properties/src/lib/property-values.component.ts

```diff
@@ -31,43 +28,15 @@ import { propertiesTypeMapping } from './resource-payloads-mapping';
     </button>`,
 })
 export class PropertyValuesComponent implements OnInit {
-  @Input() itemTpl!: TemplateRef<any>;
+  @Select(ProjectsSelectors.isCurrentProjectAdminOrSysAdmin) isCurrentProjectAdminOrSysAdmin$: Observable<boolean>;
 
-  // TODO: this is copy pasted from ProjectBase: this needs to be in StateManagement (not project base)
-  isAdmin$ = combineLatest([
-    this._store.select(UserSelectors.user).pipe(filter(user => user !== null)) as Observable<ReadUser>,
-    this._store.select(UserSelectors.userProjectAdminGroups),
-    this._route.params,
-    this._route.parent!.params,
-  ]).pipe(
-    map(([user, userProjectGroups, params, parentParams]) => {
-      // TODO this can lead to many bugs: params depend on the routing, but this component can be used everywhere!!
-      // TODO this needs to change, probably the StateManagement + Routing needs to be reviewed.
-      const uuid = ((_params: Params, _parentParams: Params) => {
-        if (_params['uuid']) {
-          return _params['uuid'];
-        }
-        if (_parentParams['uuid']) {
-          return _parentParams['uuid'];
-        }
-        if (_params['project']) {
-          return _params['project'];
-        }
-        throw new Error('Project uuid param not found');
-      })(params, parentParams);
-      const projectIri = this._projectService.uuidToIri(uuid);
-      return ProjectService.IsProjectAdminOrSysAdmin(user, userProjectGroups, projectIri);
-    })
-  );
+  @Input() itemTpl!: TemplateRef<any>;
```

Comment:
you can leave the @Input on top of the component, as it is best practice to have them on top